src/centralstore/incremental/NodeManager.cpp

```diff
@@ -143,6 +146,7 @@ RelationBlock *NodeManager::addEdge(std::pair<std::string, std::string> edge) {
 
     node_manager_logger.debug("DEBUG: Source DB block address " + std::to_string(sourceNode->addr) +
                               " Destination DB block address " + std::to_string(destNode->addr));
+    // lockEdgeAdd.unlock();
```

Comment:
Lets remove this commented line

Reply:
Remove the commented line https://github.com/miyurud/jasminegraph/pull/132/commits/d8d887f5453408266df16065d675be6691184b72

Reply:
Ok